doc/ref_model/chapters/chapter05.md

```diff
@@ -315,9 +315,9 @@ The configurations specified in here will be used in specifying the actual hardw
 
 | Reference           | Feature                                             | Description                                                        | Basic Type | Network Intensive |
 |---------------------|-----------------------------------------------------|--------------------------------------------------------------------|------------|-------------------|
-| infra.hw.cpu.cfg.001 | Number of CPU (Sockets)                             | This determines the minimum number of CPU sockets within each host | 2          | 2                 |
-| infra.hw.cpu.cfg.002 | Number of Cores per CPU                             | This determines the number of cores needed per CPU.                | 20         | 20                |
-| infra.hw.cpu.cfg.003 | NUMA                                                | NUMA support and BIOS configured to enable NUMA                    | N          | Y                 |
+| infra.hw.cpu.cfg.001 | Minimum Number of CPU (Sockets)  | This determines the minimum number of CPU sockets within each host | 2          | 2                 |
```

Comment:
```suggestion
| infra.hw.cpu.cfg.001 | Minimum Number of CPU sockets  | This determines the minimum number of CPU sockets within each host | 2          | 2                 |
```

Reply:
This was the original text -- I had only added "Minimum".